dnf5/include/dnf5/shared_options.hpp

```diff
@@ -65,10 +65,6 @@ void create_allow_downgrade_options(dnf5::Command & command);
 /// The values are stored in the `destdir` configuration option
 void create_destdir_option(dnf5::Command & command);
 
-/// Create the `--forcearch` option for a command provided as an argument.
-/// The values are stored in the `forcearch` configuration option
-void create_forcearch_option(dnf5::Command & command);
```

Comment:
It may sound bad, but this is a change of API.  What about to consider less dramatic deprecation path?

Reply:
Ah right, the command-line interface should be the same so I assume you're strictly referring to `create_forcearch_option`. Do you think it would be more appropriate to (1) make `create_forcearch_option` a no-op with a deprecation message printed as a warning or (2) throw an error here?

Reply:
I left `create_forcearch_option` as a no-op with comments, but I did not add a runtime warning.